apps/builder/app/builder/features/breakpoints/breakpoints-selector.tsx

```diff
@@ -65,7 +65,7 @@ export const BreakpointsSelector = ({
         type="single"
         value={selectedBreakpoint.id}
         onValueChange={(breakpointId: string) => {
-          if (!breakpointId.length) {
+          if (!breakpointId.length || !breakpoints.has(breakpointId)) {
```

Comment:
I'd say the second check + comment is enough.

Reply:
> I'd say the second check + comment is enough.

Got it